AIPscan/Data/report_data.py

```diff
@@ -430,10 +430,23 @@ def aips_by_puid(
     )
 
 
-def agents_transfers(storage_service_id, storage_location_id=None):
-    """Return information about agents involved in creating a transfer
-    and provide some simple statistics, e.g. ingest start time and
-    ingest finish time.
+def agents_transfers(
+    storage_service_id, start_date, end_date, storage_location_id=None
+):
+    """Return overview of transfers and their agents.
+
+    Includes simple statistics such as ingest start and finish times.
+
+    :param storage_service_id: Storage Service ID (int)
+    :param start_date: Inclusive AIP creation start date
+        (datetime.datetime object)
+    :param end_date: Inclusive AIP creation end date
+        (datetime.datetime object)
+    :param storage_location_id: Storage Location ID (int)
+
+    :returns: "report" dict containing following fields:
+        report["StorageName"]: Name of Storage Service queried
+        report["FormatVersions"]: List of result files ordered desc by size
```

Comment:
I think this help text is incorrect, the returned fields are StorageName, StorageLocation, and a list of ingests (AIPUUID, User, IngestStartDate, IngestFinishDate).  I'm not sure if this is surfaced in the UI anywhere though, or just for developers.

Reply:
you're totally right! lazy copy-paste error :woman_facepalming:

Reply:
it's also just for developers, fyi